parm/config/config.base.nco.static

```diff
@@ -49,7 +49,6 @@ export BASE_GIT="/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git"
 export DO_BUFRSND="YES"    # BUFR sounding products
 export DO_GEMPAK="YES"     # GEMPAK products
 export DO_AWIPS="YES"      # AWIPS products
-export WAFSF="YES"         # WAFS products
```

Comment:
I thought this was staying as "NO". I'm fine with removing if not.

Reply:
Even if it was `YES`, it would do nothing. So.